qa/L0_sequence_batcher/sequence_batcher_test.py

```diff
@@ -104,6 +106,59 @@ def check_deferred_exception(self):
             if len(_deferred_exceptions) > 0:
                 raise _deferred_exceptions[0]
 
+    def precreate_register_regions(self, value_list, dtype, i, batch_size=1):
```

Comment:
Are these the same as the ones in infer-utils? Can we just reuse those?

Reply:
The ones in the infer utils are specific to that case. In the sequence batcher the number of values may be variable and the function above takes care of the same to create, set and unregister all shared memory regions together to avoid any thread specific failures and make it cleaner.